src/test/java/codeu/model/data/UserTest.java

```diff
@@ -14,28 +14,30 @@
 
 package codeu.model.data;
 
+
 import java.time.Instant;
 import java.util.UUID;
+
 import org.junit.Assert;
 import org.junit.Test;
 
 public class UserTest {
 
-  @Test
-  public void testCreate() {
-    UUID id = UUID.randomUUID();
-    String name = "test_username";
-    String passwordHash = "$2a$10$bBiLUAVmUFK6Iwg5rmpBUOIBW6rIMhU1eKfi3KR60V9UXaYTwPfHy";
-    Instant creation = Instant.now();
-    Boolean admin = false;
-
-    User user = new User(id, name, passwordHash, creation, admin);
-
-    Assert.assertEquals(id, user.getId());
-    Assert.assertEquals(name, user.getName());
-    Assert.assertEquals(passwordHash, user.getPasswordHash());
-    Assert.assertEquals(creation, user.getCreationTime());
-    Assert.assertEquals(name + " hasn't written a bio yet.", user.getBio());
-    Assert.assertEquals(false, user.isAdmin());
-  }
+    @Test
+    public void testCreate() {
```

Comment:
Did you intend to change the indentation here? Please keep to 2 spaces per level